documentation/synthetic_tokens/security/security.md

```diff
@@ -0,0 +1,107 @@
+# Attack Vectors
+
+Below is a list of known attack vectors to the proper collateralization of synthetic tokens deployed using this financial contract template. 
+We’ve categorized and ranked the attack vectors by severity. 
```

Comment:
Are these ranked by severity? They appear more categorized by type.

Reply:
If you consider "severity" as whether each issue is more or less impactful to the contract, then attack vectors would appear above open issues, especially those where the conclusion is "assume assumptions not met". 

We can remove this sentence instead.